app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command_and_file_names: str) -> None:
+    source_file = command_and_file_names.split()[1]
+    new_file = command_and_file_names.split()[2]
```

Comment:
Use variable to store command_and_file_names.split() result. Validate input command, whether it is cp or not and check command lenght (should be exacty 3 arguments)